bentoml/_internal/io_descriptors/multipart.py

```diff
@@ -27,7 +29,7 @@ def predict(image: "PIL.Image.Image", annotations: t.Dict[str, t.Any]) -> t.Dict
     curl -i -F image=@test.jpg -F annotations=@test.json localhost:5000/predict
     """
 
-    def __init__(self, **inputs: t.Dict[str, IODescriptor]):
+    def __init__(self, **inputs: IODescriptor):
```

Comment:
do you mean inputs: MultiPartIO here?

Reply:
I think typehint `**` can't take a `t.Dict`

Reply:
looks like `**` just assumes all the args are the same type, so `IODescriptor` here is correct